models/clm/readme.rst

```diff
@@ -1,40 +1,195 @@
 CLM
 ===
 
-.. attention::
-
-   ``CLM`` has a private development branch with some features that are delaying its integration with the rest
-   of the public DART repository.  Until that time, you should consider this documentation as out-of-date.
-   There are several groups that are successfully using recent versions of CLM and recent versions of DART.  
-   If you are interested in using ``CLM`` with more recent versions of DART, contact DAReS staff.
-
+|CLM gridcell breakdown|                              
+Figure from the CLM5.0 Tech Note. The image is a link to the document.
```

Comment:
Please do not review the scripting section of this file.  The scripting updates for the supported release will be covered in another pull request.